client/src/views/pages/recent-bans.js

```diff
@@ -98,7 +97,10 @@ export default function () {
                         </td>
                         <td>
                           Banned on {edge.node.banList.organisation.name}'s {edge.node.banList.name}{' '}
-                          for {edge.node.reason}.
+                          <br />
+                          {edge.node.reason === 'Unknown'
+                            ? 'No reason has been specified.'
```

Comment:
I don't think this looks that great:
```
Banned on The Coalition's Public Server No reason has been specified.
```

Probably best to:
 * Replace `Unknown` with `an unknown reason` so that it reads `Banned on The Coalition's Public Server for an unknown reason.`
 * Make the `reason` field lowercase for both known and unknown reasons.
 * Add a full stop to the end of the sentence.

Might be a good idea to also add `ban list` after that ban list name so it reads: `Banned on The Coalition's Public Server ban list for...`.